controllers/devbox/internal/stat/lvm_monitor/lvm_monitor_client/lvm_monitor_client.go

```diff
@@ -0,0 +1,225 @@
+package lvmmonitorclient
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/labring/sealos/controllers/devbox/internal/stat"
+	pb "github.com/labring/sealos/controllers/devbox/internal/stat/protoc"
+	"github.com/labring/sealos/controllers/pkg/utils/logger"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/health/grpc_health_v1"
+)
+
+type LVMClient struct {
+	conn         *grpc.ClientConn
+	client       pb.LVMMonitorServiceClient
+	healthClient grpc_health_v1.HealthClient
+	address      string
+	timeout      time.Duration
+}
+
+type LVMClientConfig struct {
+	Address       string
+	Timeout       time.Duration
+	RetryCount    int
+	RetryInterval time.Duration
+}
+
+func DefaultLVMClientConfig() *LVMClientConfig {
+	return &LVMClientConfig{
+		Address:       stat.DefaultLVMClientAddr,
+		Timeout:       stat.DefaultLVMClientTimeout,
+		RetryCount:    stat.DefaultLVMClientRetryCount,
+		RetryInterval: stat.DefaultLVMClientRetryInterval,
+	}
+}
+
+// NewLVMClient creates a new LVM client
+func NewLVMClient(config *LVMClientConfig) (*LVMClient, error) {
+	if config == nil {
+		config = DefaultLVMClientConfig()
+	}
+
+	conn, err := grpc.NewClient(config.Address, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		return nil, fmt.Errorf("failed to create gRPC connection: %w", err)
+	}
+
+	client := &LVMClient{
+		conn:         conn,
+		client:       pb.NewLVMMonitorServiceClient(conn),
+		healthClient: grpc_health_v1.NewHealthClient(conn),
+		address:      config.Address,
+		timeout:      config.Timeout,
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), config.Timeout)
```

Comment:
Avoid using `context.Background()`. If you want a placeholder context, use `context.TODO()`